src/components/NotFoundPage.tsx

```diff
@@ -0,0 +1,3 @@
+export const NotFoundPage = () => (
+  <h1 className="notFound-title">Page not found</h1>
+)
```

Comment:
No redirect (required routed update to include redirecting to not found page).
No info how to go back to the home page (add text like "go back to [link to home]".